CSharp/Library/Microsoft.Bot.Connector.NetFramework/Content/Microsoft.Bot.Connector.xml

```diff
@@ -495,6 +465,17 @@
             Gets or sets the original ID this message is a response to
             </summary>
         </member>
+        <member name="P:Microsoft.Bot.Connector.Activity.Label">
+            <summary>
+            Gets or sets descriptive label
+            </summary>
+        </member>
+        <member name="P:Microsoft.Bot.Connector.Activity.ValueType">
+            <summary>
+            Gets or sets unique string which identifies the shape of the value

```

Comment:
I think this comment could be a little more explicit since it's new.  Below the same property has the comment "Unique string which identifies the format of the value object"  which I think is more descriptive.